.env.example

```diff
@@ -12,6 +13,7 @@ SECRET_ACCESS_KEY=<SECRET>
 
 # Devnet
 NETWORK=devnet
+ASSETS=SOL,BTC
```

Comment:
Yeh just update this to YAML array pls, ofc need to update on ECS once pushed too

Reply:
Following on from our chat, YAML arrays won't work here or in `aws-lambdas`:(  Our best bet is to either do `ASSETS=SOL,BTC` and string split, or `ASSETS=["SOL","BTC"]` and parse with JSON